app/graphql/types/organizations/billing_configuration.rb

```diff
@@ -6,6 +6,7 @@ class BillingConfiguration < Types::BaseObject
       graphql_name 'OrganizationBillingConfiguration'
 
       field :document_locale, String
+      field :eu_tax_management, Boolean
```

Comment:
We are trying to get rid of the "billing_configuration" as it's not a real object. Shouldn't we move the attribute directly at organization level?

Also, I would suggest to name it: `eu_customer_taxes_autodetection`